src/components/AuthProvider/AuthProvider.tsx

```diff
@@ -12,17 +13,43 @@ const initialAuthState: AuthState = {
   user: undefined,
 };
 
+const getUserType = (userGroups: Array<string>) => {
+  let userType = UserEnum.Customer;
+  userGroups.forEach((userGroup) => {
+    if (userGroup === UserEnum.Admin) userType = userGroup;
+  });
+  return userType;
+};
```

Comment:
This would be a bit easier to read I think.
```suggestion
const getUserType = (userGroups: Array<string>): UserType => {
  return userGroups.includes(UserType.Admin) ? UserType.Admin : UserType.Customer;
};
```